packages/flutter/lib/src/gestures/drag.dart

```diff
@@ -63,10 +63,22 @@ bool _isFlingGesture(Velocity velocity) {
 }
 
 abstract class _DragGestureRecognizer<T extends dynamic> extends OneSequenceGestureRecognizer {
```

Comment:
The class uses `+` on objects of type `T` and without `as dynamic` the analyzer gets sad.